app/models/load_balancer_listener.rb

```diff
@@ -15,4 +15,26 @@ class LoadBalancerListener < ApplicationRecord
   has_many :resource_groups, -> { distinct }, :through => :load_balancer_pool_members
 
   virtual_total :total_vms, :vms, :uses => :vms
+
+  def load_balancer_port_range_str
```

Comment:
Not sure I like the `_str` ending.  I generally don't like abbreviations, and it doesn't really provide intent, so perhaps it should be named `display_load_balancer_port_range`.  Alternately, if this is for display/UI purposes, perhaps it should live in the UI directly as a general presentation helper named `format_port_range(range)`, which could then be called in the UI like `format_port_range(load_balance_listener.load_balancer_port_range)`